src/test/java/seedu/medibook/logic/parser/EditNoteCommandParserTest.java

```diff
@@ -65,42 +57,18 @@ public void parse_invalidValue_failure() {
         // invalid date
         assertParseFailure(parser, "1" + INVALID_NOTE_DATE_DESC, Date.MESSAGE_CONSTRAINTS);
 
-        // invalid name
-        assertParseFailure(parser, "1" + INVALID_NOTE_DOCTOR_NAME_DESC, Name.MESSAGE_CONSTRAINTS);
-
-        // invalid mcr
-        assertParseFailure(parser, "1" + INVALID_NOTE_DOCTOR_MCR_DESC, Mcr.MESSAGE_CONSTRAINTS);
-
         // invalid content
         assertParseFailure(parser, "1" + INVALID_NOTE_CONTENT_DESC, Content.MESSAGE_CONSTRAINTS);
-
-
-        // invalid date followed by valid name
-        assertParseFailure(parser, "1" + INVALID_NOTE_DATE_DESC + NOTE_DOCTOR_NAME_DESC,
-                Date.MESSAGE_CONSTRAINTS);
-
-        // valid name followed by invalid name. The test case for invalid name followed by valid name
-        // is tested at {@code parse_invalidValueFollowedByValidValue_success()}
-        assertParseFailure(parser, "1" + NOTE_DOCTOR_NAME_DESC + INVALID_NOTE_DOCTOR_NAME_DESC,
-                Name.MESSAGE_CONSTRAINTS);
-
-        // multiple invalid values, but only the first invalid value is captured
-        assertParseFailure(parser,
-                "1" + INVALID_NOTE_DOCTOR_NAME_DESC + INVALID_NOTE_DOCTOR_MCR_DESC + NOTE_CONTENT_DESC,
-                Name.MESSAGE_CONSTRAINTS);
```

Comment:
Can these test methods be replaced using other fields? Like change name field to either date/content, so that these tests are still carried out, just on other fields that are still available.